graftm/hmmer.py

```diff
@@ -318,10 +353,10 @@ def __check_euk_contamination(self, hmm_hit_tables, run_stats):
         
         Returns
         -------
-        run_stats : dict
-            Updated form of the above.
+        euk_reads : array
```

Comment:
this line is unecessary since the name of the variable that is returned has no meaning outside this function